index.html

```diff
@@ -1,22 +1,21 @@
 <!DOCTYPE html>
 <html lang="es">
-	
 <head>
 	<meta charset="UTF-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<meta http-equiv="X-UA-Compatible" content="ie=edge">
-	<link rel="stylesheet" type="text/css" href="css/home.css">
+	<link rel="stylesheet" media="only screen and (min-width: 360px) and (max-width: 960px)" href="css/home.css">
+	<link rel="stylesheet" media="only screen and (min-width: 960px)" href="css/home.css">
 	<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
 	<title>Hostal las paisas</title>
 </head>
-
 <body>
 	<nav>
 		<!-- Navigation section -->
 	</nav>
-	<section class="homepage-hero">
+	<!--html><section class="homepage-hero">
```

Comment:
Why do you have the element <!--html>

Reply:
There is a issue with the hero, is a problem with his code , victor will fix in his PR , the hero is not let me to show my footer. and for that i disable the part of the code of him.